exercicios/para-casa/resposta-repeticao.py

```diff
@@ -0,0 +1,16 @@
+# Desenvolvendo um programa que faça a tabuada de um número qualquer. O número será o 3
+
+num1 = int(input("Digite o início: "))
+num2 = int(input("Digite o final: "))
+num3 = int(input("Tabuada para o número: "))
+sequencia = range(num1, num2 + 1)
+
+while int(num1) <= int(num2):
```

Comment:
Gostei da iniciativa de "corrigir" o erro do usuário, porém não está funcionando por alguns motivos:
A sequência é criada antes do while que incrementa a variável de início. Além disso, a variável num1 com seu novo valor não é usada após o laço while, já que no momento que entramos no laço for, num1 passa a ser cada item da sequência. 

Dá uma olhadinha no vídeo da correção que lá tem um exemplo usando while!